scripts/populate_preprint_providers.py

```diff
@@ -72,8 +74,10 @@ def update_or_create(provider_data):
 
 
 def format_domain_url(domain):
-    return ''.join((PREPRINT_PROVIDER_DOMAINS['prefix'], str(domain), PREPRINT_PROVIDER_DOMAINS['suffix'])) if \
-        PREPRINT_PROVIDER_DOMAINS['enabled'] else ''
+    prefix = PREPRINT_PROVIDER_DOMAINS['prefix'] or PROTOCOL
```

Comment:
There might be a safer way to do this with `furl` or `urlparse` but this seems safe enough for now assuming we're fairly confident in our data.